packages/terra-clinical-result/tests/wdio/clinical-result-blood-pressure-spec.js

```diff
@@ -54,4 +54,11 @@ Terra.describeViewports('ClinicalResultBloodPressure', ['medium'], () => {
 
     Terra.it.validatesElement();
   });
+
+  describe('single results', () => {
```

Comment:
Isn't this test for no results? Not single result?

Reply:
Renamed https://github.com/cerner/terra-clinical/pull/614/commits/b0b5305327e8c17a49550348602d6ea79f39c368#diff-26fedcf401089021742523790c5c19a2R1